src/rules/check-domains-against-connection-aliases.js

```diff
@@ -5,10 +5,11 @@
  * 
  * Check user email domain matches domains configured in connection
  * 
- * This rule will check that the email the user has used to login matches any of the domains configured in a connection. If there are no domains configured, it will allow access.
+ * This rule checks if the user's login email matches any domains configured in a connection. If there are no matches, the login is denied. But, if there are no domains configured it will allow access.
  * 
- * For example, to setup SAML login, a Fabrikam customer must have a managed domain (claimed and verified by the customer). Fabrikam can then enforce a policy where only users belonging to managed email domains should be able to login via SAML. For example, if the customer Contoso has setup contoso.com as a managed domain, only users with email ending @contoso.com (not @contosocorp.com) should be able to login via SAML.
- * Because Auth0 doesn't enforce this validation OOB - we have to store the valid email domain in connection object (lock already uses this) and then use a rule to validate incoming user's email domain with the one configured on the connection. If email domains doesn't match, the login is denied.
+ * Use this rule to only allow logins from users belonging to managed email domains.
```

Comment:
`managed domains` is one example use case, but not the only one. It should be clear this is an example use case.

Another sample use case: A company that has multiple forests/departments, each associated to a different domain, and only some of those forest/departments are supposed to access a particular application. In that case, the domain check rule would be akin to verifying a group or role membership, in order to provide access control.